src/lib/components/SiteFooter/index.jsx

```
@@ -3,6 +3,23 @@ import PropTypes from 'prop-types';
 import { Hyperlink, Icon } from '@edx/paragon';
 
 class SiteFooter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.externalLinkClickHandler = this.externalLinkClickHandler.bind(this);
+  }
+
+  externalLinkClickHandler(event) {
+    const label = event.currentTarget.getAttribute('href');
+    const eventName = 'edx.bi.footer.link';
+    const properties = {
+      category: 'outbound_link',
+      label,
+    };
+    if (this.props.trackEvent) {
+      this.props.trackEvent(eventName, properties);
```

Reviewers: Are we ok with silently ignoring if this isn't supplied?